commands-yml/commands/session/create.yml

```diff
@@ -6,26 +6,72 @@ description:
     The server should attempt to create a session that most closely matches the desired and required capabilities. 
     
     * [JSONWP Spec](https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#session-1) Required capabilities have higher priority than desired capabilities and must be set for the session to be created
-    * [W3C specification])(https://www.w3.org/TR/webdriver/#dfn-new-session) capabilities.alwaysMatch must be set for session to be created; capabilities.firstMatch must match at least one (the first one to match will be used)
+    * [W3C Spec](https://www.w3.org/TR/webdriver/#dfn-new-session) capabilities.alwaysMatch must be set for session to be created; capabilities.firstMatch must match at least one (the first one to match will be used)
     
 example_usage:
   java:
     |
-      MobileElement element = (MobileElement) driver.findElementByAccessibilityId("SomeAccessibilityID");
-      element.clear();
+        DesiredCapabilities desiredCapabilities = new DesiredCapabilities();
+        desiredCapabilities.setCapability(MobileCapabilityType.PLATFORM_VERSION, "10.3");
+        desiredCapabilities.setCapability(MobileCapabilityType.DEVICE_NAME, "iPhone Simulator");
+        desiredCapabilities.setCapability(MobileCapabilityType.AUTOMATION_NAME, "XCUITest");
+        desiredCapabilities.setCapability(MobileCapabilityType.APP, "/path/to/ios/app.zip");
+
+        URL url = new URL("http://127.0.0.1:4723/wd/hub");
+
+        IOSDriver driver = new IOSDriver(url, desiredCapabilities);
+        String sessionId = driver.getSessionId().toString();
+        
   python:
     |
-      self.driver.find_element_by_accessibility_id('SomeAccessibilityID').clear()
+      desired_caps = {}
```

Comment:
This can be in object literal notation like js:
```python
desired_caps = {
  'platformName': 'Android',
  'platformVersion': '7.0',
  'deviceName': 'Android Emulator',
  'automationName': 'UiAutomator2',
  'app': PATH('/path/to/app'),
}
```